packages/sage/src/commands/generate.ts

```diff
@@ -0,0 +1,77 @@
+import { Command } from 'commander'
+import { readFile, writeFile } from 'node:fs/promises'
+import { logger } from '../core/logger.js'
+import type { Manifest, CommandEntry, ContextEntry, EventConfig, CommandOption } from '@roboplay/robo.js'
+import path from 'node:path'
+
+const command = new Command('generate')
+command.command('doc').description('generates a basic doc file for the project').action(generateDocAction)
```

Comment:
`docs` is a more common abbreviation rather than `doc` (singular). Please change it to `docs`.